port/win32/omrsl.c

```diff
@@ -166,8 +166,10 @@ omrsl_open_shared_library(struct OMRPortLibrary *portLibrary, char *name, uintpt
 
 	Trc_PRT_sl_open_shared_library_Event1(openName);
 
-	/* LoadLibrary will try appending .DLL if necessary */
-	dllHandle = LoadLibraryW(unicodeName);
+	/* LoadLibraryExW will try appending .DLL if necessary
+	 * LOAD_LIBRARY_SEARCH_DLL_LOAD_DIR causes os to search for dependencies in the same folder. Simillar behaviour to LOAD_WITH_ALTERED_SEARCH_PATH.
+	 */
```

Comment:
fixes typo, less than 100 chars per line and added a period.
```suggestion
	/* LoadLibraryExW will try appending .DLL if necessary.
	 * LOAD_LIBRARY_SEARCH_DLL_LOAD_DIR causes the operating system to search for dependencies in the same folder. 
	 * This behaviour is similar to LOAD_WITH_ALTERED_SEARCH_PATH.
	 */
```